connector-sdk/feel-wrapper/src/main/java/io/camunda/connector/feel/FeelEngineWrapper.java

```diff
@@ -90,7 +88,6 @@ private static String trimExpression(final String expression) {
   private static scala.collection.immutable.Map<String, Object> toScalaMap(
       final Map<String, Object> responseMap) {
     final HashMap<String, Object> context = new HashMap<>(responseMap);
-    context.put(RESPONSE_MAP_KEY, responseMap);
```

Comment:
@sbuettner this line adds the whole `responseMap` under `response`. In the line above, the `responseMap` becomes the context itself first